README.md

```diff
@@ -41,6 +41,41 @@ If you need custom modifications to this system for your device, clone this
 repository and update as described in [Making custom
 systems](https://hexdocs.pm/nerves/customizing-systems.html).
 
+
+## Depedencies
+
+* OTP 25 or higher
+* libmnl-dev or equivilent 
```

Comment:
`libmnl-dev` shouldn't be needed. Did you get an error?

Reply:
Yeah, in order to build the firmware I had to do it.

Reply:
I believe it was around `nerves_system` if I remember correctly. I can try to re-clean and confirm

Reply:
after cleaning it several times and removing, I can't replicate the error. will remove that for now.